logprep/connector/http/input.py

```diff
@@ -390,15 +274,33 @@ class Config(Input.Config):
             validator=[
                 validators.instance_of(dict),
                 validators.deep_mapping(
-                    key_validator=validators.in_(UVICORN_CONFIG_KEYS),
-                    # lamba xyz tuple necessary because of input structure
+                    key_validator=validators.in_(http.UVICORN_CONFIG_KEYS),
+                    # lambda xyz tuple necessary because of input structure
                     value_validator=lambda x, y, z: True,
                 ),
             ]
         )
 
         """Configure uvicorn server. For possible settings see
         `uvicorn settings page <https://www.uvicorn.org/settings>`_.
+
+        .. security-best-practice::
+           :title: Uvicorn Webserver Configuration
+           :location: uvicorn_config
+           :suggested-value: below
+           
+           Additionaly to the below it is recommended to configure `ssl on the metrics server endpoint
+           <https://www.uvicorn.org/settings/#https>`_
+
+           .. code-block:: yaml
+               :caption: Recommended uvicorn configuration for connector endpoint
```

Comment:
Removing the caption will solve the sphinx warning. Apparently there is an internal shpinx issue when adding code blocks with captions inside the security best practices.
```suggestion
```